### PR TITLE
Redesign Audio Icon on the StyledCard

### LINKED ISSUE #232: Audio Icon Change
**Goals**

Before:
Screen Shot 2022-10-13 at 12 24 48 AM

After:
Screen Shot 2022-10-13 at 12 23 52 AM

Change the Audio Icon on the `LessonHome` page to the new icon on Figma. You will also need to change the positioning of the icon on the `<StyledCard/>`.

### PR DESCRIPTION
**Pull request by rustom**
## Description

Restyles the StyledCard to match new figma design. Works on both mobile and tablet. 

Fixes #232.

### CODE REVIEW

File: client/src/components/StyledCard/StyledCard.js [hunk 1]
```diff
@@ -9,36 +9,49 @@ import { INDICATOR_TYPES } from 'utils/constants'
...
+    paddingRight: 20,
   },
   soundIcon: {
     alignItems: 'center',
     justifyContent: 'center',
-    marginLeft: 10,
-  },
-  indicator: {
-    marginRight: 15,
+    backgroundColor: '#dddddd',
```
Comment (ashayp22) [inline client/src/components/StyledCard/StyledCard.js:49]: Use `#DEE5E9` (from the figma)

File: client/src/components/StyledCard/StyledCard.js [hunk 2]
```diff
@@ -9,36 +9,49 @@ import { INDICATOR_TYPES } from 'utils/constants'
...
   },
   left: {
+    display: 'flex',
     flexDirection: 'row',
-    justifyContent: 'space-between',
+    flexShrink: 2,
+  },
+  middle: {
+    paddingLeft: 15,
+    flexShrink: 2,
```
Comment (ashayp22) [inline client/src/components/StyledCard/StyledCard.js:25]: What is the flexShrink used for?

**Review by ashayp22 — CHANGES_REQUESTED** [review]
Looks good, left two inline comments

File: client/src/components/StyledCard/StyledCard.js (hunk 1 above)
Comment (rustom) [reply client/src/components/StyledCard/StyledCard.js]: done

File: client/src/components/StyledCard/StyledCard.js (hunk 2 above)
Comment (rustom) [reply client/src/components/StyledCard/StyledCard.js:25]: to make sure that the text component shrinks dynamically based on the size of the button box and image box, lmk if it doesn't work properly

**Review by ashayp22 — APPROVED** [review]

**Pull request merged**